figo/figo.py

```diff
@@ -129,20 +129,24 @@ class FigoException(Exception):
     They consist of a code-like `error` and a human readable `error_description`.
     """
 
-    def __init__(self, error, error_description):
+    def __init__(self, error, error_description, code):
```

Comment:
Following the approach of #33, I'll set the default of `code` to `None`. As I stated in one of my initial commits, I extended the http specific error messages hardcoded in `figo.py` by an error code of 90000, indicating a unspecified error, if I remember correctly.

What's your take on keeping it this way for consistency reasons? In case you're happy with the status quo, I'll remove it. 🙂